src/main/java/org/mitre/synthea/world/agents/Clinician.java

```diff
@@ -0,0 +1,105 @@
+package org.mitre.synthea.world.agents;
+
+import com.google.common.collect.HashBasedTable;     
+import com.google.common.collect.Table;              
+import com.google.gson.internal.LinkedTreeMap;       
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Random;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.apache.sis.geometry.DirectPosition2D;
+import org.apache.sis.index.tree.QuadTree;
+import org.apache.sis.index.tree.QuadTreeData;
+import org.mitre.synthea.engine.EventList;
+import org.mitre.synthea.world.concepts.HealthRecord;
+import org.mitre.synthea.world.concepts.VitalSign;
+
+
+
+
+
+public class Clinician implements Serializable, QuadTreeData {
```

Comment:
Because java doesn't support multiple inheritance, we need to decide whether it makes more sense for `Clinician` to `extend Person` or `extend Provider` or neither.

Extending `Person` provides all sorts of demographics and the ability to generate medical records for the clinicians themselves. While extending `Clinician` allows us to track how many procedures a doctor performed, for example. A third option would be to simply contain `Person` and `Provider` objects and use them appropriately. We need to keep in mind that Clinicians will be able to work at multiple `Provider` organizations. We may want to separate the `Interface` from the classes, so we can reuse them.